vignettes/Overview.Rmd

```diff
@@ -0,0 +1,448 @@
+|                              | `FastqSampler`       | Draw random samples from FASTQ files       |
+|                              | `readFastq`          | Read an entire FASTQ file into memory      |
+|                              | `writeFastq`         | Write FASTQ objects to a connection (file) |
+|                              | `countFastq`         | Quickly count FASTQ records in files       |
+| **Sequence and quality summary** |                    |                                            |
+|                              | `alphabetFrequency`  | Nucleotide or quality score use per read   |
+|                              | `alphabetByCycle`    | Nucleotide or quality score use by cycle   |
+|                              | `alphabetScore`      | Whole-read quality summary                 |
+|                              | `encoding`           | Character / ‘phred’ score mapping          |
+| **Quality assessment**           |                    |                                            |
+|                              | `qa`                 | Visit FASTQ files to collect QA statistics |
+|                              | `report`             | Generate a quality assessment report       |
+| **Filtering and trimming**       |                    |                                            |
+|                              | `srFilter`           | Pre-defined and bespoke filters            |
+|                              | `trimTails`, etc.    | Trim low-quality nucleotides               |
+|                              | `narrow`             | Remove leading / trailing nucleotides      |
+|                              | `tables`             | Summarize read occurrence                  |
+|                              | `srduplicated`, etc. | Identify duplicate reads                   |
+|                              | `filterFastq`        | Filter reads from one file to another      |
+
+: (\#tab:table) Key functions for working with FASTQ files
+
+```{r countFastq}
+fl <- system.file(package="ShortRead", "extdata", "E-MTAB-1147", 
+                  "ERR127302_1_subset.fastq.gz")
+countFastq(fl)
+```
+
+Small FASTQ files can be
+read in to memory in their entirety using `readFastq`; we do
+this for our sample data
+
+```{r readFastq}
+fq <- readFastq(fl)
+```
+
+The result of data input is an instance of class `ShortReadQ`
+
+|                    |                                                   |
+| ------------------ | ------------------------------------------------- |
+| DNAStringSet       | (`r Biocpkg("Biostrings")`) Short read sequences |
+| FastqQuality, etc. | Quality encodings                                 |
+| ShortReadQ         | Reads, quality scores, and ids                    |
+
+**Table 2:** Primary data types in the `r Biocpkg("ShortRead")` package.
+
+This class contains reads, their quality scores, and optionally the id
+of the read.

```

Comment:
I looked at this section again and I think the .Rnw splits the short paragraph. It's also missing the link to table 2, which should be like table 1. Could you change this section to

```
The result of data input is an instance of class `ShortReadQ` 
(Table \@ref(tab:table2)). This class contains reads, their quality scores, and
optionally the id of the read.

|                    |                                                   |
| ------------------ | ------------------------------------------------- |
| DNAStringSet       | (`r Biocpkg("Biostrings")`) Short read sequences  |
| FastqQuality, etc. | Quality encodings                                 |
| ShortReadQ         | Reads, quality scores, and ids                    |

: (\#tab:table2) Primary data types in the `r Biocpkg("ShortRead")` package.

```

Reply:
Done.

Reply:
The paragraphs were split to match the formatting of the PDF document. If you find that there are more paragraphs that needs to be fixed please let me know.